doc/legal/legal.md

```diff
@@ -38,7 +38,6 @@ general this code is available under these licences:
 * Eclipse Public License version 1.0, or
 * GNU General Public License version 2, or
 * GNU Lesser General Public License version 2.1, or
-* Common Public License 1.0 (only for the parser)
 
 See `epl-1.0.txt`, `gpl-2.txt`, `lgpl-2.1.txt`, `cpl-1.0.txt`.
```

Comment:
I think you can also delete `cpl-1.0.txt` now if nothing else in this file refers to it, and you can remove `cpl-1.0.txt` in this line. Can you also remove those lines about the licenses mix up if the POM is now fixed.